src/components/Calculator.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+export default function Calculator() {
+  return (
+    <>
+      <div className="container">
+        <input type="text" className="resultBar" placeholder="0" />
+        <div className="keypad">
+          <button type="button" className="singleButton">AC</button>
+          <button type="button" className="singleButton">+/-</button>
+          <button type="button" className="singleButton">%</button>
+          <button type="button" className="singleButton">/</button>
+          <button type="button" className="singleButton">7</button>
+          <button type="button" className="singleButton">8</button>
+          <button type="button" className="singleButton">9</button>
+          <button type="button" className="singleButton">X</button>
+          <button type="button" className="singleButton">4</button>
+          <button type="button" className="singleButton">5</button>
+          <button type="button" className="singleButton">6</button>
+          <button type="button" className="singleButton">-</button>
+          <button type="button" className="singleButton">1</button>
+          <button type="button" className="singleButton">2</button>
+          <button type="button" className="singleButton">3</button>
+          <button type="button" className="singleButton">+</button>
+          <button type="button" className="doubleButton">0</button>
+          <button type="button" className="singleButton">.</button>
+          <button type="button" className="singleButton">=</button>
+        </div>
+      </div>
+    </>
+  );
+}
```

Comment:
- [ ] Please make sure to create at least one child component for the Calculator component as requested in the project requirements. (I would suggest separating the buttons code in a different component and importing it here in the Calculator component).

![image](https://user-images.githubusercontent.com/83803414/234105771-d4c9e88d-2e24-4490-9bae-563b9fbdc2e0.png)